src/Schema/TableDiff.php

```diff
@@ -56,6 +56,18 @@ public function __construct(
             );
         }
 
+        foreach ($droppedForeignKeys as $droppedForeignKey) {
+            $constraintName = $droppedForeignKey->getObjectName()?->toString();
+            if ($constraintName === null || $constraintName === '') {
```

Comment:
We cannot use `getObjectName()` in the library code in 4.4.x since it can throw an exception, if the name cannot be parsed using the 5.0.x logic but can be using the 4.4.x one. Please use `if ($droppedForeignKey->getName() === '')`.